src/builtins/builtins_exit.c

```diff
@@ -0,0 +1,58 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   builtins_exit.c                                    :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: yliu <yliu@student.42.jp>                  +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2024/09/06 17:47:26 by yliu              #+#    #+#             */
+/*   Updated: 2024/09/17 19:27:01 by yliu             ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "builtins.h"
+#include <errno.h>
+
+static bool	is_numeric(const char *exit_status)
```

Comment:
The functionality of this function is provided by EINVAL and the endptr parameter of ft_strtol.

e.g.1 not numeric
`ft_strtol("abc", &endptr, 10);`
=> errno is EINVAL

e.g.2 not numeric
`ft_strtol("123abc", &endptr, 10);`
=> `endptr` points 'a'

e.g. 3 numeric
`ft_strtol("123", &endptr, 10);`
=> `endptr` points '\0'